perfiles_usuario/tests.py

```diff
@@ -1,3 +1,145 @@
-# from django.test import TestCase
+from django.test import TestCase
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import Group
 
-# Create your tests here.
+from .utils import is_member, is_administrador, is_capturista, is_directivo, is_servicios_escolares
+from .utils import ADMINISTRADOR_GROUP, CAPTURISTA_GROUP, \
+                DIRECTIVO_GROUP, SERVICIOS_ESCOLARES_GROUP
+from .models import Capturista
+
+
+class PermisosTestCase(TestCase):
```

Comment:
Documentation for the class is needed. The method setUp also needs documentation.